crd/multitenancy/api/v1alpha1/nodeinfo.go

```diff
@@ -36,6 +38,17 @@ type NodeInfoSpec struct {
 	VMUniqueID string `json:"vmUniqueID,omitempty"`
 }
 
+// NodeInfoStatus defines the observed state of NodeInfo
+type NodeInfoStatus struct {
```

Comment:
I believe Status should be a subresource, check out NodeNetworkConfig for an example of the annotation

Reply:
Added subresource annotation.